conf/config.json

```diff
@@ -201,7 +201,8 @@
                     "bindAddress": "localhost",
                     "port": 4045
                 }
-            }
+            },
+            "coldStorageArchiveTopicPrefix": "cold-archive-req-"
```

Comment:
this should be set in `lifecycle`?
or it could be defined in constants instead

Reply:
I had it in constants, Rached suggested to have this in config. At the moment, it is being used by ReplicationAPI, should this still be in lifecycle?

Reply:
it could possibly be set at the root of the config object. 
I believe the `joi` validator will need to be updated as well

Reply:
moved it under `lifecycle`.

Reply:
yes. it should be validated.